ui/nuxeo-document-permissions/nuxeo-document-acl-table.js

```diff
@@ -122,7 +123,9 @@ import './nuxeo-popup-permission.js';
                         [[entityDisplay(ace.username)]]
                       </span>
                     </div>
-                    <div class="flex-2"><span class="label">[[i18n(ace.permission)]]</span></div>
+                    <div class="flex-2">
+                      <span class="label">[[formatPermission(ace.permission)]]</span>
```

Comment:
We need this to be recomputed if the language chages:
```suggestion
                      <span class="label">[[formatPermission(ace.permission, i18n)]]</span>
```